test/format/ops.c

```diff
@@ -500,6 +500,9 @@ operations(u_int ops_seconds, u_int run_current, u_int run_total)
      */
     rollback_to_stable(session);
 
+    if (GV(DISAGG_MULTI_CHECKSUM))
```

Comment:
Could we call it `MULTI_VALIDATE`? From the Users perspective, this would mean validate databases, and as part of it we will computing checksum.

Reply:
I disagree that this should be separate. It has constraints that only make it realistic as part of disagg, e.g. the requirement that it's only used for row-store, the requirement for predictable replay, and the incompatibility with the `timer` config.